doc/man3/EC_GROUP_new.pod

```diff
@@ -135,17 +116,17 @@ relevant equation introduced above.
 
 EC_group_get_curve() obtains the previously set curve parameters.
 
-EC_GROUP_set_curve_GFp() and EC_GROUP_set_curve_GF2m() are synonyms for
+EC_GROUP_set_curve_GFp() is a synonym for
```

Comment:
i think the docs usually call these 'aliases' (there are more, which i won't flag)